tests/clvm/coin_store.py

```diff
@@ -70,7 +70,8 @@ def validate_spend_bundle(
         if result.error is not None:
             raise BadSpendBundleError(f"condition validation failure {Err(result.error)}")
 
-        ephemeral_db = dict(self._db)
+        ephemeral_db = dict(self._db.items())
```

Comment:
whats going on here?, was this wrong ?
i dont see any changes to what we take out of ephemeral_db and i assume the tests passed before

Reply:
Creating a dict from a dict like it was before does what you might expect.  Creating a dict from an iterable of two-tuples, such as from `.items()`, uses the first element of each tuple as the key and the second as the value.  So the code does the same thing.  Still leaves open the question of what this has to do with the rest of the PR.

Reply:
so if it does the same thing why the change ?
also yes not related to this pr

Reply:
You are both correct - this crept in when I was slimming this PR down. I'll revert this change, of course.